app/main.py

```diff
@@ -2,0 +1,36 @@
+from os import path, remove, makedirs
+
+
+def move_file(command: str) -> None:
+    command_words = command.split(" ")
+
+    if len(command_words) != 3 or command_words[0] != "mv":
```

Comment:
The condition correctly checks for the command format as per the checklist requirement: 'Make sure you check that the input string contains 3 elements separated by space, and has `mv` on a first place.'